app/models/manageiq/providers/ansible_tower/inventory/collectors/automation_manager.rb

```diff
@@ -0,0 +1,19 @@
+class ManageIQ::Providers::AnsibleTower::Inventory::Collectors::AutomationManager < ManagerRefresh::Inventory::Collector
+  def inventories
+    manager.with_provider_connection do |connection|
+      connection.api.inventories.all
+    end
+  end
+
+  def hosts
+    manager.with_provider_connection do |connection|
+      connection.api.hosts.all
+    end
+  end
+
+  def job_templates
+    manager.with_provider_connection do |connection|
```

Comment:
makes sense, and it was like this before.

is this a stable API? Is `with_provider_connection` avail in every manager? Maybe we can hide this from the user somehow...

But falling back to keeping the connection for now